proj/src/main/webapp/script.js

```diff
@@ -50,17 +50,85 @@ function errorCallback(err){
 function getLocation() {
     if(navigator.geolocation){
         navigator.geolocation.getCurrentPosition(successCallback,errorCallback,{timeout:10000, enableHighAccuracy:false});
-    }
-    else{
+    } else{
         console.log("Error. Geolocation not supported or not enabled");
     }
     return;
 }
 
+/*
+    Setting password 
+*/ 
+
+// Check if the password meets the password requirement 
+$('#pass').on('keyup', function(){
+    const password = document.getElementById("pass");
+    var goodPassword = false;
+    passwordLength(password, goodPassword);
+    containsNumber(password, goodPassword);
+    containsSymbol(password, goodPassword);
```

Comment:
Did you mean to return goodPassword from these three functions and do something like `goodPassword &=passwordLength(password)`

Reply:
Alternatively you can do something like
`var goodPassword = [passwordLength,containsNumber,containsSymbol].every(function(handler){return handler(password)})` if you want to get fancy.

Reply:
haha definitely going to implement the fancy one